tools/gen_internal_conf/src/main.rs

```diff
@@ -793,6 +793,8 @@ struct OcclumMountOptions {
     pub temporary: bool,
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub cache_size: Option<String>,
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub disk_size: Option<String>,
```

Comment:
Need to check `disksize` validity.(`disksize > 5 * GB`)

Reply:
Decide to check `disk_size.is_none()` at first moment we got Ext2's mount config, and leave `disk_size < 5 * GB` at `SwornDiskMeta::setup()`.